lib/bundler.rb

```diff
@@ -444,7 +444,7 @@ def load_gemspec(file, validate = false)
 
     def load_gemspec_uncached(file, validate = false)
       path = Pathname.new(file)
-      contents = read_file(file)
+      contents = path.read(:encoding => Encoding::UTF_8)
```

Comment:
It's surprising Bundler still supports the very old Ruby 1.8.7.
Maybe `File.open(file, "r:UTF-8", &:read)` would work on 1.8.7 too.

Reply:
Thanks for the info, I agree in general. I filed https://github.com/bundler/bundler/issues/6600 just before seeing this, let's see.

Reply:
It didn't work... :disappointed: Up to the maintainers how to follow up from here.